src/openforms/forms/models/form.py

```diff
@@ -146,6 +146,14 @@ class Form(models.Model):
             "Whether the step progression should be displayed in the UI or not."
         ),
     )
+    show_summary_progress = models.BooleanField(
+        _("show summary of the progress"),
+        default=False,
+        help_text=_(
+            "Whether the summary of the progress should be displayed in the UI or not. "
+            "If enabled the user will be able to see the the steps progression of the form."
+        ),
```

Comment:
```suggestion
        help_text=_(
            "Whether to display the short progress summary, indicating the current step "
            "number and total amount of steps."
        ),
```
and please use the same help text in the JS literals